lib/robots/sdr_repo/preservation_ingest/transfer_object.rb

```diff
@@ -97,6 +105,19 @@ def transfer_bag
           end
         end
 
+        def transfer_bag_via_scp
+          Open3.popen2e(scp_command) do |_stdin, stdout_and_stderr, wait_thr|
+            output = stdout_and_stderr.read
+            status = wait_thr.value
+
+            raise "Transfering bag for #{druid} to preservation failed. STDOUT = #{output}" if status.nil? || !status.success?
+          end
+        end
+
+        def scp_command
+          "scp -pr #{from_host}:#{from_dir}/#{bare_druid} #{deposit_dir}"
```

Comment:
We can also add the `-q` flag to make scp print less stuff once we're sure this works as expected. That should make the command run a bit faster too.